### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by monicasarbu**
Add new feature: "Publish expired DNS transactions"

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ All notable changes to this project will be documented in this file based on the
 ### Added
 - add [.editorconfig file](http://editorconfig.org/)
 - add (experimental/unsupported?) saltstack files
+- Publish expired DNS transactions
```
Comment (ruflin) [inline CHANGELOG.md]: I would suggest to add the line only once, either under Added or Improvements.

**Comment by ruflin** [comment]
@urso @monicasarbu It would be nice to link the related pull request in the future at the end of the line of a comment (in case there is a specific one).

**Comment by urso** [comment]
@ruflin chicken egg problem? Who's first the PR id to link to or the commit :)

File: CHANGELOG.md (hunk 1 above)
Comment (urso) [inline CHANGELOG.md]: +1 for only once. I would consider this a bug fix.

**Comment by ruflin** [comment]
@urso Yep. But this is solved in most as we have the "pull request" as conversation an open it very early.

File: CHANGELOG.md (hunk 1 above)
Comment (tsg) [inline CHANGELOG.md]: Why do we have both Added and Improvements, anyway? On http://keepachangelog.com/ I don't see Improvements.

Comment (urso) [inline CHANGELOG.md]: Good point. Just stumbled upon this too. +1 For removing 'Improvements'.

**Comment by urso** [comment]
merge conflicts

**Comment by monicasarbu** [comment]
Ready for review.

**Comment by monicasarbu** [comment]
Delete in favor of https://github.com/elastic/packetbeat/pull/311.

**Pull request closed without merging**